content/learn/book/welcome/app/_index.md

```diff
@@ -0,0 +1,82 @@
++++
+title = "Creating Bevy apps"
+weight = 2
+template = "book-section.html"
+page_template = "book-section.html"
++++
+
+Bevy programs store and execute all of their game logic and data with a single `App` data structure.
+Let's make a trivial Hello World app.
+
+```rust
+use bevy::prelude::*;
+
+fn main(){
+  App::new()
+  .add_system(hello.system())
+  .run();
+}
+
+fn hello(){
+    println!("Hello, Bevy!")
+}
+```
+
+The process is straightforward: we first create a new [`App`](https://docs.rs/bevy/latest/bevy/app/struct.App.html).
+Then, we add a simple system, which prints "Hello, Bevy!" when it is run.
+Finally once we're done configuring the app, we call [`App::run()`](https://docs.rs/bevy/latest/bevy/app/struct.App.html#method.run) to actually make our app *do things*.
+
+## What makes an App?
+
+So, what sort of data does are `App` really store?
+Turns out: it's fairly simple.
+Looking at the docs for [`App`](https://docs.rs/bevy/latest/bevy/app/struct.App.html), we find three fields: `world`, `runner` and `schedule`.
+You can read more about these by exploring the reference documentation linked just above.
+
+Generally, you'll be operating at a higher-level of abstraction than these basic primitives: thinking about data in terms of specific resources or components, adding systems via convenient methods rather than manually supplying a `Schedule`, and sticking to the default runner which runs your game logic in a loop.
```

Comment:
higher-level -> higher level

Reply:
```suggestion
Generally, you'll be operating at a higher-level of abstraction. You'll likely be thinking about data in terms of specific resources or components, or adding game logic systems via methods to be run in the [default runner](?url?).
```

Also, is "default runner" a Bevy specific term? Are there docs for this to be linked to?
url?? == https://docs.rs/bevy/0.5.0/bevy/app/struct.App.html#structfield.runner

Reply:
I think the default runner talk is mostly just confusing. I cut it.